packages/core/src/elements/ResponsiveElement.ts

```diff
@@ -30,7 +30,15 @@ export type ResizeEvent = CustomEvent<{
  * @returns {void}
  */
 const triggerResize = (entry: ResizeObserverEntry): void => {
-  const { inlineSize: width, blockSize: height } = entry.borderBoxSize[0];
+  let width; let height;
```

Comment:
This should be better:
```suggestion
  let width: number;
  let height: number;
```

Reply:
Fixed!